tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -108,6 +114,248 @@ public function test_create_checkout_session(
+			throw $e;
+		} finally {
+			$_POST = $post_before;
+			if ( null !== $test_request ) {
+				remove_filter( 'pre_http_request', $test_request, 10, 3 );
+			}
+			Ajax_Test_Helper::remove_hooks();
+		}
+
+		$response = json_decode( $output );
+		$this->assertEquals( (object) $expected_response, $response );
+	}
```

PHPMD (2.15.0)

[error] test_update_checkout_session accesses the super-global variable $_POST. (undefined)

(Superglobals)